.github/workflows/changesets-master.yml

```diff
@@ -26,18 +35,58 @@ jobs:
 
       - name: Install Dependencies
         run: yarn
-      
+
       - name: Configure git
         run: |
           git config --global user.name 'Neo4j Team GraphQL'
           git config --global user.email 'team-graphql@neotechnology.com'
 
-      - name: Create Release Pull Request
+      - name: Creating .npmrc
+        run: |
+          cat << EOF > "$HOME/.npmrc"
+            @neo4j:registry https://registry.npmjs.org
+            //registry.npmjs.org/:_authToken=$NPM_TOKEN
+          EOF
+        env:
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: Create release PR or release
```

Comment:
Should that be "for release", or am I misunderstanding how this is doing two different things?

Reply:
Yeah this will either create the release PR (i.e. https://github.com/neo4j/graphql/pull/1742), _or_ it will perform the release itself, when the aforementioned PR is merged. Bit of a confusing one! Happy to accept alternative naming suggestions.